content/blog/2021-06-16-june-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,242 @@
+
+> I have recently came across with DVC by listening to MLOps Coffee Sessions #6
+> with David Aponte and Elle O'Brien (Such an interesting talk! 💯). This tool
+> integrates smoothly with Git, tracks models & datasets, and also has an online
+> UI DVC Studio 🚀. Is there any use case of MLflow that DVC can't handle? I
+> find DVC to give more rise to creativity as it integrates really well with
+> Git. - André Godinho
+
+## Neda Sultova Tutorial
+
+Drilling down to the next level, I give you
+[this tutorial](https://medium.com/geekculture/exploring-dvc-for-machine-learning-pipelines-in-research-part-1-3ebc2ca35a18)
+by [Neda Sultova](https://www.linkedin.com/in/neda-sultova-597a811a8/). Not only
+is it a great tutorial of DVC in and of itself, but Neda started with her use
+case and the framework for deciding on what tools will bridge the gaps and
+increase efficiency for her team at [Helmholtz AI](https://www.helmholtz.ai/).
+Among the needs are reproducibility, workflow integration, exchangeable backend,
+framework agnostic, open source and the ability of the solution to be tweaked to
+the team's needs.
+
+<external-link
+href="https://medium.com/geekculture/exploring-dvc-for-machine-learning-pipelines-in-research-part-1-3ebc2ca35a18"
+title="Exploring DVC for Machine Learning Pipelines in Research (Part 1)"
+description="The first of a multi-part series on the search and decision making process for MLOps tools at Helmholtz AI."
+link="https://medium.com"
+image="/uploads/images/2021-06-15/neda-sultova.png"/>
+
+## Our Philosophy
+
+And at last I bring you to
+["The Road to AI Hell Starts with Good MLOps Intentions" ](https://thenewstack.io/the-road-to-ai-hell-starts-with-good-mlops-intentions/)
+by our CEO [Dmitry Petrov](https://twitter.com/FullStackML) which explains our
+philosophy in the MLOps space. You will learn about Dmitry's experience that led
+to our tools, what we think is the right way to solve MLOps challenges, and how
+we do it.
+
+> Teams made up of data scientists and developers should be able to define their
+> own workflow based on their business requirements and team preferences, just
+> like they do today when constructing any other software artifact. Rather than
+> a platform forcing teams to embrace a highly opinionated workflow, they can
+> employ flexible tools such Git, GitHub, and their existing CI tools as they
+> see fit. - Dmitry Petrov
+
+<external-link
+href="https://thenewstack.io/the-road-to-ai-hell-starts-with-good-mlops-intentions/"
+title="The Road to AI Hell Starts with Good MLOps Intentions"
+description="Dmitry Petrov explains the journey and philosophy at the heart of Iterative.ai's MLOps tools."
+link="https://thenewstack.io"
+image="/uploads/images/2021-06-15/ai-hell.png"/>
+
+# Big News! 🚀🚀🚀
+
+In case you missed it, June 3rd we introduced our latest tool that offers a GUI
+to display your team's work with DVC and CML: DVC Studio! We know this has been
```

Comment:
```suggestion
In case you missed it, June 3rd we introduced our latest tool: [DVC Studio](https://studio.iterative.ai/)! A web application which GUI
displays your team's work with DVC and CML We know this has been
```
> Needs wrapping